### PR TITLE
refs #14 - Manage topic name for manager and step

### PR DESCRIPTION
**Pull request by mventuravoyageprive**
#14

### CODE REVIEW

File: Library/Services/TopicService.php [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+ *
+ * (c) Matthieu Ventura <mventura@voyageprive.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Vpg\Disturb\Services;
+
+class TopicService
```
Comment (jrmbrgs) [inline Library/Services/TopicService.php:13]: shall we suffix service classes by `Service`

File: Library/Services/TopicService.php [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+     * Generate formated topic name for workflow manager
+     *
+     * Example : disturb-LOAD_CONTRACT_28_FR-manager
+     *
+     * @param String $workflowName Workflow Name
+     *
+     * @throws \Exception Required parameters
+     * @return String
+     */
+    public static function getWorkflowManagerTopicName (string $workflowName) : string
```
Comment (jrmbrgs) [inline Library/Services/TopicService.php:35]: not sure we need `Workflow` in the func name

File: Library/Services/TopicService.php [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+     *
+     * Example : disturb-step0-LOAD_CONTRACT_28_FR-step
+     *
+     * @param String $stepName     Step Name
+     * @param String $workflowName Workflow Name
+     *
+     * @throws \Exception Required parameters
+     * @return String
+     */
+    public static function getWorkflowStepTopicName (string $stepName, string $workflowName) : string
```
Comment (jrmbrgs) [inline Library/Services/TopicService.php:59]: the same as above

**Review by jrmbrgs — CHANGES_REQUESTED** [review]

File: Library/Services/TopicService.php (hunk 1 above)
Comment (mventuravoyageprive) [reply Library/Services/TopicService.php:13]: #30

File: Library/Services/TopicService.php (hunk 2 above)
Comment (mventuravoyageprive) [reply Library/Services/TopicService.php:35]: #30

File: Library/Services/TopicService.php (hunk 3 above)
Comment (mventuravoyageprive) [reply Library/Services/TopicService.php:59]: #30

**Pull request merged**